data/Templates/Hl7v2/ADT_A01.liquid

```diff
@@ -5,9 +5,6 @@
         {% assign firstSegments = hl7v2Data | get_first_segments: 'PID|PD1|PV1|PV2|AVR|MSH' -%}
 
         {% evaluate messageHeaderId using 'ID/MessageHeader' MSH: firstSegments.MSH -%}
-        {% evaluate organizationId4 using 'ID/Organization' HD: firstSegments.MSH.4 -%}
-        {% evaluate organizationId6 using 'ID/Organization' HD: firstSegments.MSH.6 -%}
-        {% evaluate organizationId22 using 'ID/Organization' XON: firstSegments.MSH.22 -%}
```

Comment:
Are we removing organizations from MSH segment?

Reply:
Yes, the `MSH` and its related resources(especially Organization) seems contain some issues in mapping tables. So we should remove them currently and fix them in `v3.4` or later releases.